js_modules/dagit/packages/core/src/ui/Filters/useTimeRangeFilter.tsx

```diff
@@ -67,10 +67,12 @@ type Args = {
 export function useTimeRangeFilter({
   name,
   icon,
-  timezone,
+  timezone: _timezone,
   initialState,
   onStateChanged,
 }: Args): TimeRangeFilter {
+  const timezone =
+    _timezone === 'Automatic' ? Intl.DateTimeFormat().resolvedOptions().timeZone : _timezone;
```

Comment:
If it's all the same I'd vote to put the useContext(TimeContext) in here to eliminate the possibility of another callsite doing this incorrectly in the future. I'm not sure there's a use case where the timezone should /not/ be the one from the time context?

Reply:
If it's `Automatic`, it should use `browserTimezone()`

Reply:
Ah in the Audit log we explicitly use UTC, I think its intentional but I'm in favor of being consistent here so lets change it and see if anyone complains